app/actions/remote/post.ts

```diff
@@ -435,6 +436,26 @@ export async function fetchPosts(serverUrl: string, channelId: string, page = 0,
     }
 }
 
+export async function refreshPosts(serverUrl: string, channelId: string) {
+    try {
+        const fetched = await fetchPosts(serverUrl, channelId);
+        if (fetched.error) {
+            throw fetched.error;
+        }
+
+        const lastFetchedAt = getLastFetchedAtFromPosts(fetched.posts);
```

Comment:
This may generate some problems. Imagine the following scenario (not sure how real):
- We are missing 70 messages on the channel.
- We refresh the posts.
- fetchPosts get the last 60 messages.
- There is a gap with 10 missing messages

Another potential problem...
- I edit a post of a month ago, but not received the event for whatever reason.
- I refresh the posts
- I get the posts, and the latest post is after the edit.
- I don't receive the edit, because I am getting the posts, not the posts since
- Since I set the fetchAt based on the posts, and there is a post after the edit, I never receive the edit.

Just to make sure, I would let the refresh refresh the posts, but not update the lastFetchedAt.

Reply:
Ok I can revert this back to how it was which basically won't update the last fetched at